addons/stock/report/report_stock_reception.xml

```diff
@@ -0,0 +1,179 @@
+                        <span t-esc="reason"/>
+                    </t>
+                </h3>
+            </div>
+            <div><table t-if="sources_to_lines" class="o_report_reception_table table table-sm">
+                <t t-foreach="sources_to_lines" t-as="source">
+                    <thead t-if="report_type == 'html' or any(line['is_assigned'] for line in sources_to_lines[source])">
+                        <tr class="bg-light">
+                            <th>
+                                <i t-if="source.priority == '1'" class="o_priority o_priority_star fa fa-star"/>
+                                <a name="source_link" href="#" t-att-res-model="source._name" view-type="form" t-att-res-id="source.id" t-esc="source.display_name"/>
+                                <span t-if="source._name == 'stock.picking' and source.origin">
+                                    (<t t-esc="source.origin"/>)
+                                </span>
+                                <span t-if="source._name == 'stock.picking' and source.partner_id">:
+                                    <a name="source_link" href="#"
+                                        t-att-res-model="source.partner_id._name"
+                                        t-att-res-id="source.partner_id.id"
+                                        view-type="form"
+                                        t-esc="source.partner_id.name"/>
+                                </span>
+                            </th>
+                            <th>Expected Delivery: <t t-esc="sources_to_formatted_scheduled_date[source]"/></th>
+                            <th t-if="report_type == 'html' and any(s['move_ins'] for s in sources_to_lines[source])">
+                                <button t-if="any(not s['is_assigned'] and s['is_qty_assignable'] for s in sources_to_lines[source])"
+                                        class="btn btn-sm btn-primary o_report_reception_assign o_assign_all"
+                                        name="assign_source_link">
+                                        Assign All
+                                </button>
+                            </th>
+                            <th t-if="report_type == 'html'">
+                                <button t-if="any(s['is_assigned'] for s in sources_to_lines[source])"
+                                        class="btn btn-sm btn-primary o_print_label o_print_label_all"
+                                        name="print_labels">
+                                        Print Labels
+                                </button>
+                                <button t-elif="any(s['move_ins'] for s in sources_to_lines[source])"
+                                        class="btn btn-sm btn-primary o_print_label o_print_label_all"
+                                        name="print_labels" disabled="1">
+                                        Print Labels
+                                </button>
```

Comment:
Same than the other print button `disabled` (I hope I didn't mix up the conditions)
```suggestion
                                <button t-if="any(s['move_ins'] for s in sources_to_lines[source])"
                                        class="btn btn-sm btn-primary o_print_label o_print_label_all"
                                        t-att-disabled="not any(s['is_assigned'] for s in sources_to_lines[source])"
                                        name="print_labels">
                                        Print Labels
                                </button>
```